packages/scss/src/components/inputs/textfield/_textfield.compact.scss

```diff
@@ -165,27 +100,13 @@
 		}
 	}
 
-	&.mod-withSuffix {
-		.textfield-input {
-			padding-right: _component("textfield.suffix.padding-right");
-		}
-	}
-
 
 	// STATES
 	// ▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒
 
 	// Disabled & readonly
 	.textfield-input {
-		&[disabled], &[readonly] {
-			background: _theme("commons.background.dark");
-			background-image: none;
-			background-position: bottom;
-		}
-
 		&.is-error {
```

Comment:
aren't states specific declarations unecessary now ?

Reply:
As we increase specificity with mod-compact, error state also need to have a bigger weight